docs/pages/docs/handbook/publishing-packages/versioning-and-publishing.mdx

```diff
@@ -0,0 +1,50 @@
+# Versioning and Publishing Packages in a Monorepo
+
+Manually versioning and publishing packages in a monorepo can be extremely tiresome. Luckily, there's a tool that makes things easy - the [Changesets](https://github.com/changesets/changesets) CLI.
+
+We recommend Changesets because it's intuitive to use, and - just like Turborepo - fits with the monorepo tools you're already used to.
+
+Some alternatives are:
+
+- [intuit/auto](https://github.com/intuit/auto) - Generate releases based on semantic version labels on pull requests
+- [microsoft/beachball](https://github.com/microsoft/beachball) - The Sunniest Semantic Version Bumper
+
+## Understanding Changesets
+
+We recommend taking a look at the changesets docs. Here's our recommended reading order:
+
+1. [Why use changesets?](https://github.com/changesets/changesets/blob/main/docs/intro-to-using-changesets.md) - an intro that takes you through the fundamentals.
+1. [Installation instructions](https://github.com/changesets/changesets/blob/main/packages/cli/README.md)
+1. If you're using GitHub, consider using the [Changeset GitHub bot](https://github.com/apps/changeset-bot) - a bot to nudge you to add changesets to PR's.
+1. You should also consider adding the [Changesets GitHub action](https://github.com/changesets/action) - a tool which makes publishing extremely easy.
+
+## Using Changesets with Turborepo
+
+Once you've started using changesets, you'll gain access to three useful commands:
+
+```bash
+# Add a new changeset
+changeset
+
+# Create new versions of packages
+changeset version
+
+# Publish all changed packages to npm
+changeset publish
+```
+
+Linking your publishing flow into Turborepo can make organising your deploy a lot simpler and faster.
+
+Our recommendation is to add a `publish-packages` script into your root `package.json`:
```

Comment:
I like this usage of "Our recommendation" throughout. Makes the difference between fact and opinion clear and teaches me where I _could_ diverge from these docs / dig deeper if I wanted to.